docs/index.js

```diff
@@ -1,7 +1,7 @@
 //---------------Smooth Scrolling---------------//
 $(function() {
 	$('a[href*="#"]:not([href="#"])').click(function() {
```

Comment:
Ok, I thought adding the # fixed it, but I was wrong. I took that out, add what you suggested above, and it seems to be working just fine now. So thanks!!